react/components/Tabs/Tab.js

```diff
@@ -16,8 +16,8 @@ class Tab extends Component {
           fullWidth ? 'w-100' : ''
         } ${
           active
-            ? 'c-on-muted b--emphasis'
-            : 'c-muted-1 b--transparent hover-c-action-primary pointer'
+            ? 'c-on-muted b--emphasis vtex-tab__button--selected'
```

Comment:
Shouldn't it be `vtex-tab__button--active` & `vtex-tab__button--inactive`?

Reply:
Might be a better name, thanks